schemas/context/enduserids.schema.json

```diff
@@ -9,42 +9,30 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "End User IDs",
   "type": "object",
-  "description": "Condensed, normalized encapsulation of all end user identifiers. NOTE: At least one of the fields is required.",
-  "minProperties": 1,
+  "description": "Condensed, normalized encapsulation of all end user identifiers. NOTE: At least one of the fields is required.\n",
   "properties": {
-    "xdm:realm": {
-      "title": "Realm",
-      "$ref": "https://ns.adobe.com/xdm/data/datasource",
-      "description": "The optional realm associated with the identity stitching strategy."
+    "https://ns.adobe.com/experience/mcId": {
+      "title": "Adobe Marketing Cloud Id",
+      "$ref": "https://ns.adobe.com/xdm/context/identity",
+      "description": "A unique identifier from Adobe Marketing Cloud."
```

Comment:
Clarify what "from" means: Generated by? Understood by? All of the above?

Reply:
From as "generated by".